### DISCUSSION TITLE
Make osu editor acceseble in Tablet

### DISCUSSION THREAD

**Discussion by futag** [ppy/osu#30292] [Ideas]
- Add delete tool aside select tool for easiest deleting object in tablet (because can't access middle click mouse)
- Add double touch for undo and tripple touch for redo (just like ibis paint)

**Comment by peppy** [comment]
Does hold for menu not work?

**Reply by futag** [reply]
But maybe edit beatmaps more easier and faster on any device (like mouse and touchscreen)
